ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Pipelines.resource

```diff
@@ -108,10 +109,10 @@ Create Pipeline Run    # robocop: disable
 Click Action From Actions Menu
     [Documentation]    Click an action from Actions menu (3-dots menu on the right)
     [Arguments]    ${pipeline_name}    ${action}
-    Wait Until Page Contains Element    xpath://table//tr[td[*[a[text()='${pipeline_name}']]]]//td[contains(@class, 'pf-c-table__action')]/div
-    Click Element    xpath://table//tr[td[*[a[text()='${pipeline_name}']]]]//td[contains(@class, 'pf-c-table__action')]/div
-    Wait Until Page Contains Element    //table//tr[td[*[a[text()='${pipeline_name}']]]]//td[contains(@class, 'pf-c-table__action')]/div/ul/li/button[contains(text(), '${action}')]
-    Click Element    //table//tr[td[*[a[text()='${pipeline_name}']]]]//td[contains(@class, 'pf-c-table__action')]/div/ul/li/button[contains(text(), '${action}')]
+    Wait Until Page Contains Element    xpath://table//tr[td[.//a[.//text()="${pipeline_name}"]]]//button[@aria-label="Kebab toggle"]
+    Click Element    xpath://table//tr[td[.//a[.//text()="${pipeline_name}"]]]//button[@aria-label="Kebab toggle"]
+    Wait Until Page Contains Element    //table//tr[td[*[a[.//text()='${pipeline_name}']]]]//td[contains(@class, '-table__action')]//div/ul/li/button[.//*[contains(text(), '${action}')]]
+    Click Element    //table//tr[td[*[a[.//text()='${pipeline_name}']]]]//td[contains(@class, '-table__action')]//div/ul/li/button[.//*[contains(text(), '${action}')]]
```

Comment:
@jiridanek would you mind trying using the same keyword from `ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot` now that https://github.com/red-hat-data-services/ods-ci/pull/1117 has been merged?

Reply:
on it

Reply:
just need to rerun the CI

Reply:
ok thanks. If it works, please delete the keyword from Pipelines resource file as well, so we clean it. I kind of remember that pipelines needed some ad-hoc change in the kw

Reply:
does not work, my table is missing `@data-label="Name"`, so I can use similar query to yours, but not the exact one ;(

Reply:
oh there it is the difference I couldn't remember..What a pity. We could ask UI team to fill the gap in the DOM